airflow/providers/trino/hooks/trino.py

```diff
@@ -15,7 +15,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-import os
+import os, ast
```

Comment:
```suggestion
import ast
import os
```

Please consider [setting up pre-commit](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#pre-commit-hooks) locally to catch issues like this.

Reply:
@uranusjr Understood. Sorry, my bad I did not set up pre-commit, I will make changes and it set it up and push again.